src/main/java/igrad/logic/commands/SelectAvatarCommand.java

```diff
@@ -11,7 +11,8 @@
  */
 public class SelectAvatarCommand extends Command {
 
-    public static final String MESSAGE_SUCCESS = "You have chosen an animal guide!";
+    public final String MESSAGE_SUCCESS;
```

Comment:
non-final message shouldn't be snake case all caps! can consider making it camel case non-final or removing this field completely

Reply:
oh I've changed it back

Reply:
actually @dargohzy, you could also declare placeholders in final strings, I'm changing this line to;
`public final String MESSAGE_SUCCESS = "Hi, I'm %s, let's get started!\n" + MESSAGE_ADD_COURSE;`